src/main/java/org/xembly/Directives.java

```diff
@@ -254,6 +254,17 @@ public Directives append(final Iterable<Directive> dirs) {
         return this;
     }
 
+    /**
+     * Appends the {@link Node node}.
+     * @param node The node to append
+     * @return This object
+     * @see #append(java.lang.Iterable)
+     * @see Directives#copyOf(org.w3c.dom.Node)
```

Comment:
@llorllale it's a good practice to add `@since` to methods we introduce. The next version will be 0.23

Reply:
@yegor256 ok